celer/lasso_fast.pyx

```diff
@@ -113,29 +115,37 @@ def celer(
         if t != 0:
             create_dual_pt(pb, n_samples, &theta[0], &Xw[0], &y[0])
 
-            scal = dnorm_l1(
-                is_sparse, theta, X, X_data, X_indices, X_indptr, screened,
-                X_mean, weights, center, positive)
+            scal = dnorm_enet(
```

Comment:
This would be the occasion to improve naming: scal is not the scaling (it used to be), it's the dual norm. 
what we call tmp later is the real scaling.

tmp is acceptable as name only when it's used over a very short number of lines, so if we can avoid using it here, it'd be better

Reply:
Should we apply it here or make it as separate PR?

Reply:
For a future PR (#247)